lms/static/js/courseware/link_clicked_events.js

```diff
@@ -0,0 +1,16 @@
+;(function(define) {
```

Comment:
This is because the JS files get concatenated together, and so the semicolon ensures that the code doesn't accidentally become part of the last function from the previous file. Once we switch over to a more modern bundling approach this won't be necessary.

http://stackoverflow.com/questions/7145514/whats-the-purpose-of-starting-semi-colon-at-beginning-of-javascript